src/main/java/com/google/alpollo/model/SongSentiment.java

```diff
@@ -10,10 +10,13 @@
   private final float score;
   /** Each sentiment has a magnitude, representing how strong the sentiment is, ranging from 0.0 to +inf. */
   private final float magnitude;
+  /** Based on the score and magnitude of a sentiment, we can make up an interpretation of these values. */
+  private final String interpretation;
 
-  public SongSentiment(float score, float magnitude) {
+  public SongSentiment(float score, float magnitude, String interpretation) {
```

Comment:
Since the interpretation is not independent of the score and magnitude, I think it would make more sense to compute it inside the constructor instead of having it as a separate argument.